Analysis/HiggsTauTau/scripts/makeDatacardsRun2.py

```diff
@@ -553,6 +596,14 @@ def split_callback(option, opt, value, parser):
         ' --background_scheme=%(bkg_scheme)s --signal_scheme=%(sig_scheme)s'
         ' --x_axis_label="%(xlab)s" --y_axis_label="dN/dm_{#tau#tau} [1/GeV]"'
         ' --blind=%(BLIND)s --x_blind_min=%(blind_min)s --x_blind_max=%(blind_max)s --verbosity=0'
+        ' --paramfile=%(PARAMS)s --extra_pad=0.2 --folder=%(FOLDER)s %(extra)s' % vars())
+    
+    print ('$CMSSW_BASE/src/UserCode/ICHiggsTauTau/Analysis/HiggsTauTau/bin/HiggsTauTauPlot5 --cfg=%(CFG)s --channel=%(ch)s'
+        ' --method=%(cat_num)s --cat=%(cat_str)s --datacard=%(dc)s'
+        ' --var="%(var)s%(bin)s" --norm_bins=true '
+        ' --background_scheme=%(bkg_scheme)s --signal_scheme=%(sig_scheme)s'
+        ' --x_axis_label="%(xlab)s" --y_axis_label="dN/dm_{#tau#tau} [1/GeV]"'
+        ' --blind=%(BLIND)s --x_blind_min=%(blind_min)s --x_blind_max=%(blind_max)s --verbosity=0'
```

Comment:
it's not duplicated it's just printing the command. I think this was for debugging when we were urgently trying to produce a data card and run expected limit in time for presentation in HTT meeting but can be removed now